client/src/Components/LinkItem/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Link from '@material-ui/core/Link';
+import useStyles from './style';
+
+function LinkItem(props) {
+  const { text, linkUrl, bgFilled } = props;
+  const classes = useStyles();
+  return (
+    <>
+      <Link
+        underline="none"
+        href={linkUrl}
+        className={`${classes.root} ${bgFilled ? classes.signupLink : ''}`}
+      >
+        {text}
```

Comment:
replace the text with {props.children} if we have to pass an icon to our LinkItem? we need to pass a node to our component